File: 26/26.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        uniqIndex = 0
        for i in range(1, len(nums)):
            if nums[i] != nums[uniqIndex]:
                nums[uniqIndex+1] = nums[i]
                uniqIndex += 1
        return len(nums[:uniqIndex+1])

File: two-pointers/26/26.js
var removeDuplicates = function(nums) {
  let uniqueIndex = 0;
  for (let i = 1; i < nums.length; i++) {
    if (nums[i] !== nums[uniqueIndex]) {
      nums[uniqueIndex + 1] = nums[i];
      uniqueIndex++;
    }
  }
  nums.splice(uniqueIndex + 1);
  return nums.length;
};
